vmm/src/config.rs

```diff
@@ -592,6 +620,13 @@ impl NetConfig {
             return Err(Error::ParseNetVhostSocketRequired);
         }
 
+        // In case the tap interface already exists, check if the number of
+        // queues is appropriate. The tap might not support multiqueue while
+        // the number of queues indicates the user expects multiple queues, or
+        // on the contrary, the tap might support multiqueue while the number
+        // of queues indicates the user doesn't expect multiple queues.
+        check_mq_support(&tap, num_queues)?;
```

Comment:
This check won't work when using vhost-user-net backend.

Reply:
what do you mean? Since we're not providing tap to vhost-user-net?

Reply:
Does this function handle "--net-backend" too ?